public/stylesheets/weekly-review.css

```diff
@@ -0,0 +1,77 @@
+.weekly-summary-modal {
+  display: none;
+  font-family: 'Open Sans', Arial;
+  background-color: white;
+  width: 370px;
+  height: 430px;
+  position:absolute;
+  left: 50%;
+  top: 50%;
+  margin-left: -200px;
```

Comment:
I don't understand the math then - the width of the container is 370px...  I would also consider the use of flex to make this much more concise:
```css
.modal-wrapper {
  flex: 1;
  width: 100%;
  min-height: 100%;
  position: absolute;
  top: 0;
  left: 0;
  justify-content: center;
  align-items: center;
}
```
Then the child container can just identify that it is flex, set its height and width, and it should be centered in the parent container, without this math.